js/ingredients.js

```diff
@@ -0,0 +1,163 @@
+// commented out filters
+    // "Name": "name",
+    // "Display Name": "displayName",
+    // "Tier": "stars",
+    // "Powder Slots": "slots",
+    // "Health": "hp",
+    // "Raw Fire Defense": "fDef",
+    // "Raw Water Defense": "wDef",
+    //"Raw Air Defense": "aDef",
+    // "Raw Thunder Defense": "tDef",
+    // "Raw Earth Defense": "eDef",
+
+    const translate_mappings = {
```

Comment:
grrr
they aren't quite the same are they

Reply:
Not quite